### ISSUE TITLE
🐛 [BUG] - Rating of some exercises do not appear in the historic

### ISSUE THREAD

**Issue by TheSecondWing** [madmustachecompany/Blast-Workout-App#72]
### Description

Hello,
Some exercises have their rating (6 to 10) not displayed in this historic of the exercise. I have verified this for planks for example.
Thanks.

![Screenshot_20240123-172424](https://github.com/madmustachecompany/Blast-Workout-App/assets/156791196/89c97b42-45a9-41d0-9c51-7a252e83047b)
![Screenshot_20240123-172412](https://github.com/madmustachecompany/Blast-Workout-App/assets/156791196/1f874cac-25c4-4fac-945a-d914a2e29525)

### Reproduction steps

```bash
1. Go to '...'
2. Tap on '....'
3. Scroll down to '....'
4. See error
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### OS

None

**Comment by madmustachecompany** [comment]
good point 👀 
should be easy to fix thanks :)

**Comment by TheSecondWing** [comment]
Thanks a lot!

**Comment by TheSecondWing** [comment]
Hello,

This problem seems to have been fixed with the newest release 1.61.

Thanks!
